src/components/ha-camera-stream.ts

```diff
@@ -163,6 +184,26 @@ class HaCameraStream extends LitElement {
     }
   }
 
+  private async _renderHLSExoPlayer(url: string) {
+    this._resizeExoPlayerListener = () => this._resizeExoPlayer();
```

Comment:
This is not necessary, see https://github.com/home-assistant/frontend/pull/6606/files#r473917697